extensions/websockets-next/deployment/src/main/java/io/quarkus/websockets/next/deployment/WebSocketProcessor.java

```diff
@@ -1382,6 +1465,10 @@ private static ExecutionModel executionModel(MethodInfo method, TransformedAnnot
     }
 
     static boolean hasBlockingSignature(MethodInfo method) {
+        if (KotlinUtils.isKotlinSuspendMethod(method)) {
```

Comment:
Does it make sense to support a kotlin suspend method annotated with `@Blocking` or `@RunOnVirtualThread`?

Reply:
Humm. I don't think so, but we should probably check that. Let me think about that a bit.

Reply:
If it does not make sense then we should probably fail the build during validation...

Reply:
Yeah, that's what I thought. I'll do that.

Reply:
Done.